filebeat/module/greenplum/log/_meta/fields.yml

```diff
@@ -0,0 +1,133 @@
+- name: log
+  description: Please add description
```

Comment:
Thanks for contributing! Could you change the description here for each field please? example is not required but would be good to have.

Reply:
For the example, should I include how it looks in the log, or how it looks after it is parsed?

Reply:
How it looks after it is parsed. Think on this as a way to document what the users will see in the final events. As Kaiyan said, it is not mandatory to add examples to all fields, but it is helpful in some cases to complement the description.

Reply:
Alright, added some examples!